example/index.tsx

```diff
@@ -22,6 +22,7 @@ class Example extends React.Component {
                     content={this.renderContent}
                     onBeforePrint={this.handleBeforePrint}
                     onAfterPrint={this.handleAfterPrint}
+                    removeAfterPrint={true}
```

Comment:
Nit: no need to be explicit when setting a bool
```suggestion
                    removeAfterPrint
```

Reply:
Done.